sysext.just

```diff
@@ -150,6 +153,14 @@ download-rpms target:
     mkdir rpms
     cd rpms
 
+    disablerepos=""
+    if [[ -n "{{disable_repos}}" ]]; then
+        for r in {{disable_repos}}; do
+            echo "➖ Disabling repo: ${r}"
+            disablerepos+=" --disablerepo=${r}"
+        done
+    fi
+
```

Comment:
Let's move that right below the "enable repos" code above.

Reply:
Ahahaha okay!